src/vic3/effect_validation.rs

```diff
@@ -555,18 +556,77 @@ pub fn validate_create_military_formation(
 
 pub fn validate_create_pop(
     _key: &Token,
-    _block: &Block,
+    block: &Block,
     _data: &Everything,
     _sc: &mut ScopeContext,
     mut vd: Validator,
     _tooltipped: Tooltipped,
 ) {
     // This effect is undocumented
 
-    vd.field_item("culture", Item::Culture);
-    vd.field_item("religion", Item::Religion);
     vd.field_item("pop_type", Item::PopType);
     vd.field_integer("size");
+
+    // TODO: Consider doing sanity checks on the numbers in the splits
+    //  Since they're percentages of the size, not ratios, if they don't add up to 1 you'll get unexpected results
+    //  But that would involve float maths and comparisons for something that isn't technically wrong.
+
+    let mut available_cultures = HashSet::new();
+
+    if block.has_key("culture") {
+        vd.ban_field("cultures", || "pops with several cultures");
+        vd.field_item("culture", Item::Culture);
+        if let Some(token) = vd.field_value("culture") {
```

Comment:
Would be simpler to get the token by using `if let Some(token) = block.get_field_value("culture")` above instead of just `has_key`

Reply:
I was under the incorrect assumption that would complain if `culture` didn't exist. I'll do that then, the old solution felt a bit dirty.